frontend/src/utils/string.js

```diff
@@ -13,3 +11,27 @@ export const extractRepoNameFromUrl = (url) => {
   }
   return url;
 };
+
+export const getPlatformUrl = ({ platform, username }) => {
+  if (platform === 'devto') {
```

Comment:
I would move this to the integration configuration

Reply:
Very good point! Thanks for the suggestion. Already updated in my last commit